packages/odyssey-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -55,7 +55,9 @@ let Tooltip = forwardRef<HTMLElement, TooltipProps>((props, ref) => {
         className={tooltipClasses}
         role="tooltip"
       >
-        {label}
+        <Text color="bodyInverse" size="caption" weight="bold">
```

Comment:
These styles should be removed from the module, right?

Reply:
Good catch, I accidentally stashed this work and didn't commit/push it up. Fixing.